### PR TITLE
Basic spec setup

### PR DESCRIPTION
**Pull request by petems**
Just runs a basic sanity test on `#initialize` for now

Sets up ladle but doesn't test it for now.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,32 @@
+GEM
```
Comment (hunner) [inline Gemfile.lock]: Could you not commit the .lock file?

**Comment by hunner** [comment]
Want to add a .travis.yml too? :)

**Comment by nibalizer** [comment]
+1

File: Gemfile.lock (hunk 1 above)
Comment (petems) [inline Gemfile.lock]: Removed and added to .gitignore :+1:

**Comment by petems** [comment]
@hunner Added basic .travis.yml, I think when I can get the ladle stuff working so we can actually test against a real LDAP server, we can have a matrix to test against multiple gem versions for hiera and net-ldap gem :+1: 

But I'll put that in the next PR! I've got a few refactoring things I wanna do but I dont want to do them until I can prove it doesnt break things :smile:

**Pull request merged**